### DISCUSSION TITLE
Will exporting an iModel mesh to 3DFT improve iTwin viewer performance?

### DISCUSSION THREAD

**Discussion by ilyasbelfar** [iTwin/itwinjs-core#6984] [Q&A] [answered]
I am currently working with the mesh export API to export big size iModels to 3DFT formats. And I have a few questions about this API:

- Will exporting an iModel mesh to 3DFT improve the iTwin viewer's performance when loading the model?
- If yes, how can I use the exported mesh to achieve this performance improvement inside the iTwin viewer?

Any guidance or examples would be greatly appreciated!

**Comment by markschlosseratbentley** [comment]
@danieliborra

**Reply by danieliborra** [reply, marked as answer]
Hello,
thank you for your questions.

> Will exporting an iModel mesh to 3DFT improve the iTwin viewer's performance when loading the model?

No, 3DFT is an experimental technology that targets Unreal users only.

> Any guidance or examples would be greatly appreciated!

I'd suggest to contact your account manager (or similar point of contact at Bentley), as we're doing some developments in that direction that could be of your interest.
